# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/IonTypes.h
# Commit: 5445f480023e
# Full Hash: 5445f480023e36e3dd1cea7bd659202ac83d25bc
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-09-09 15:22:41
# Regressor Bug: 1720422
# File Overlap Count: 2
# Description:
#   Bug 1720422 - Move js::Scalar -> JS::Scalar for public API, but alias it as js::Scalar r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D120292
# ==============================================================================

diff -r 988eccbeb2ae -r 5445f480023e js/src/jit/IonTypes.h
--- a/js/src/jit/IonTypes.h	Wed Sep 08 23:46:59 2021 +0000
+++ b/js/src/jit/IonTypes.h	Wed Sep 08 23:47:00 2021 +0000
@@ -14,6 +14,7 @@
 #include <stdint.h>
 
 #include "jstypes.h"
+#include "NamespaceImports.h"
 
 #include "js/ScalarType.h"  // js::Scalar::Type
 #include "js/Value.h"